### ISSUE TITLE
Max Kill  ?

### ISSUE THREAD

**Issue by ghost** [Minifixio/pocketmine-1vs1-plugin#25]
Just another suggestion .. You must reach the max kill to win the Match .. Maybe up to 10 Kill per Match ..

**Comment by idobyte** [comment]
hmm that sounds cool

**Comment by idobyte** [comment]
_like 1v1 on cod but you go to 9 on there the 10th is a trickshot  :)_

**Comment by Minifixio** [comment]
Ehhh sorry but I don't really understand the principle. Do you have something like a video from a server or something like this for show me how it's working. What I understood, that is like a killstreak to 10 and then the player win the match.

**Comment by ghost** [comment]
Max Score is 10 .
The First player get 10 Kill will win the match.

Can i post a link of other online games for example?

**Comment by idobyte** [comment]
okay @Minifixio what we mean is the first to get 10 kills wins like i kill you, you respawn back into the match and continue till someone get 10 kills

**Comment by ghost** [comment]
Yeah thats right

**Comment by ghost** [comment]
Coders! Please Focus on this :) This is great for Pvp Battle

**Comment by ghost** [comment]
@Minifixio  are you working in this?

**Comment by bjorguv** [comment]
@imjayl he said he was working on teams if you want this feature rushed then do it yourself

**Comment by Minifixio** [comment]
Yes I'm working on but I have school as well :) But it's almost finish, I hope the base will be finish next week-end :)

**Comment by Minifixio** [comment]
You going to have work too because there are the messages to write :)

**Comment by robske110** [comment]
? Work

**Comment by ghost** [comment]
@robske110 dude i need your help. Where can i contact you.

**Comment by robske110** [comment]
Pocketmine forums? Or on Gitter...

**Comment by ghost** [comment]
@Minifixio . ill wait for that

**Comment by Minifixio** [comment]
I hope you will help me to write the messages x)

**Comment by Minifixio** [comment]
You can contact me on Twitter too.

**Comment by robske110** [comment]
I can help :)

**Comment by robske110** [comment]
(Messages)

**Comment by ghost** [comment]
Me Too.

**Comment by ghost** [comment]
I dont have Twitter . do you have Facebook?

**Comment by Landy005** [comment]
yall sould download twitter
